storage/tokudb/ha_tokudb.cc

```diff
@@ -3695,6 +3695,7 @@ int ha_tokudb::do_uniqueness_checks(uchar* record, DB_TXN* txn, THD* thd) {
     // first do uniqueness checks
     //
     if (share->has_unique_keys && do_unique_checks(thd, in_rpl_write_rows)) {
+        DBUG_EXECUTE_IF("tokudb_rpl_do_unique_checks", DBUG_SUICIDE(););
```

Comment:
Maybe DBUG_ASSERT_UNREACHABLE or DBUG_ASSERT(0) to make it clear that this is not a crash injection site but a bug catcher (and also so that failure results in a stacktrace)